src/test/java/org/gaul/s3proxy/S3AwsSdkTest.java

```diff
@@ -253,6 +274,39 @@ public void testAwsV2UrlSigning() throws Exception {
         }
     }
 
+    @Test
+    public void testAwsV2UrlSigningWithOverrideParameters() throws Exception {
```

Comment:
Does this actually give some coverage that `testAwsV2SignatureWithOverrideParameters` does not?  I usually rely on [s3-tests](https://github.com/ceph/s3-tests) for this kind of exhaustive testing.